google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -286,6 +286,9 @@ def publish(
         Add the given message to this object; this will cause it to be
         published once the batch either has enough messages or a sufficient
         period of time has elapsed.
+        
```

Comment:
Remove white space to fix `google/cloud/pubsub_v1/publisher/client.py:289:1: W293 blank line contains whitespace`.